sdk/nodejs/api/client.gen.ts

```diff
@@ -495,9 +586,10 @@ The command being executed WILL BE GRANTED FULL ACCESS TO YOUR HOST FILESYSTEM.
    * Writes the container as an OCI tarball to the destination file path on the host for the specified platformVariants.
    * Return true on success.
    * @param path Host's destination path.
-Path can be relative to the engine's workdir or absolute.
+   * Path can be relative to the engine's workdir or absolute.
+   * @example ./tarball
```

Comment:
However, for class methods it's displayed correctly 😃 
Could we have a more explicit example though ?
```ts
/**
* @example
* export("./tarball")
*/
```

Before:
![image](https://user-images.githubusercontent.com/1186424/212037270-fbce8cc5-cc9c-4333-aef2-c74224ffa278.png)

After:
![image](https://user-images.githubusercontent.com/1186424/212037311-8d928a2b-2ed7-46a8-b713-726261e6a467.png)

Reply:
Function call example instead of parameter example, I understand!

Reply:
Fixed in https://github.com/dagger/dagger/pull/4367/commits/2a0ebdce4d802f8666d5963ca8963641169fd915

Reply:
Much better thanks 😄 ! However it display many example blocks now instead of one grouping snippets. Let move forward though and addressed that in another issue. Thanks

![image](https://user-images.githubusercontent.com/1186424/212274263-f21ddd1a-aba5-4933-a5a9-481d434b8841.png)